source/org/zfin/ontology/presentation/TermAPIController.java

```diff
@@ -226,7 +227,11 @@ public JsonResultResponse<ChebiPhenotypeDisplay> getPhenotypeChebi(@PathVariable
         pagination.addToFilterMapIfNotEmpty("chebiPhenotype.amelioratedExacerbatedPhenoSearch", filterModification);
         pagination.addToFilterMapIfNotEmpty("chebiPhenotype.fish.name", filterFishName);
         pagination.addToFilterMapIfNotEmpty("chebiPhenotype.phenotypeStatementSearch", filterPhenotype);
-        pagination.addToFilterMapIfNotEmpty("chebiPhenotype.expConditionChebiSearch", filterTermName);
+        if (isEqe) {
+            pagination.addToFilterMapIfNotEmpty("chebiPhenotype.phenotypeStatementSearch", filterTermName);
```

Comment:
Does this override the statement on line 229?

Reply:
Yes, under 'usual' conditions search in chebiPhenotype.expConditionChebiSearch while when the endpoint is called with EQE=true then search in field chebiPhenotype.phenotypeStatementSearch